### ISSUE TITLE
Unable To Create Conda Repositories with CLI

### ISSUE THREAD

**Issue by sjqnn** [jfrog/jfrog-cli#2414]
### Describe the bug

We're unable to use the cli to create new conda repositories. When trying to do so we are met with a `unsupported package type: conda` error.  Is this supported or not? We were hoping to be able to manage and update repos in this manner. Using the API the same repo config json appears to work.

### Current behavior

Cli throws error `unsupported package type: conda`

### Reproduction steps

```
$ cat conda.json

{"key":"conda-local","packageType":"conda","rclass":"federated","repoLayoutRef":"simple-default"}

$ jf rt rc conda.json --access-token "MY-ACCESS-TOKEN" --url https://myartifactoryinstance/artifactory

19:54:35 [🚨Error] unsupported package type: conda
```

### Expected behavior

Conda Repository to Be Created

### JFrog CLI version

2.52.4

### Operating system type and version

Alpine Linux v3.19

### JFrog Artifactory version

7.71.8

**Comment by sverdlov93** [comment]
Hi @sjqnn 
Thanks for reporting this issue.
Our apologies for the inconvenience caused by this.
This [PR](https://github.com/jfrog/jfrog-cli-core/pull/1106) should fix it.
I will update this thread once the next release will be out.

**Comment by sverdlov93** [comment]
Hi @sjqnn 
We believe that the issue was fixed on JFrog CLI 2.52.10 so feel free to update the cli version to the latest.
Looking forward to your feedback on it.
